it/uk/gov/hmrc/fhregistrationfrontend/controllers/ContactPersonControllerISpec.scala

```diff
@@ -0,0 +1,258 @@
+package uk.gov.hmrc.fhregistrationfrontend.controllers
+
+import org.jsoup.Jsoup
+import play.api.libs.ws.DefaultWSCookie
+import play.api.test.WsTestClient
+import uk.gov.hmrc.fhregistrationfrontend.testsupport.{Specifications, TestConfiguration}
+
+class ContactPersonControllerISpec
+  extends Specifications with TestConfiguration {
+
+  "GET /fhdds/contactPerson" should {
+
+    "render the Contact Person's Details page" in {
+      given
+        .commonPrecondition
+
+      WsTestClient.withClient { client =>
+        val result = client.url(s"$baseUrl/contactPerson ")
+          .addCookies(DefaultWSCookie("mdtp", authAndSessionCookie))
+          .get()
+
+        whenReady(result) { res =>
+          res.status mustBe 200
+          val page = Jsoup.parse(res.body)
+          page.title() must include("Contact person's details")
+          page.getElementsByTag("h1").text() must include("Contact person's details")
+        }
+      }
+    }
+  }
+
+  "POST /fhdds/contactPerson" when {
+    "details are entered and Yes radio button selected" should {
+      "return 200 with contact person's details" in {
+        given
+          .commonPrecondition
+
+        WsTestClient.withClient { client =>
+          val result = client.url(s"$baseUrl/contactPerson ")
+            .addCookies(DefaultWSCookie("mdtp", authAndSessionCookie))
+            .withHttpHeaders(xSessionId,
+              "Csrf-Token" -> "nocheck")
+            .post(Map(
+              "firstName"               -> Seq("John"),
+              "lastName"                -> Seq("Smith"),
+              "jobTitle"                -> Seq("Astronaut"),
+              "telephone"               -> Seq("0123456789"),
+              "usingSameContactAddress" -> Seq("true")
+            ))
+
+          whenReady(result) { res =>
+            res.status mustBe 200
+            res.body mustBe "Form submitted, with result: ContactPerson(John,Smith,Astronaut,0123456789,None,true,None,None,None)"
+          }
+        }
+      }
+    }
+
```

Comment:
Add new lines to make more readable ? aka: 

"details are entered," +
      "Is this the address you want to use? No \n" +
      "Is the contact address in the UK? Yes \n" +

Reply:
New line characters added to test descriptions in latest comment.